### PR TITLE
CAMT-607 shutter sound while capturing photo

### PR DESCRIPTION
**Pull request by saurabhathavale**

### CODE REVIEW

**Comment by Sean7852** [comment]
Looks like you messed up this PR with the flash PR

File: app/src/main/java/com/leia/leiacam2/CameraFragment.kt [hunk 1]
```diff
@@ -160,6 +180,39 @@ class CameraFragment : Fragment(), CaptureInterface, SwitchInterface, CameraMode
...
+            mShutterSoundPlayer?.release()
+        }
+    }
+    override fun onDestroy() {
+        super.onDestroy()
+        when (mAudioManager?.ringerMode) {
+            AudioManager.RINGER_MODE_NORMAL -> {
+                mAudioManager?.setStreamVolume(AudioManager.STREAM_MUSIC, mDefaultVolume, AudioManager.FLAG_PLAY_SOUND)
+            }
+        }
```
Comment (Sean7852) [inline app/src/main/java/com/leia/leiacam2/CameraFragment.kt:209]: What does this do?

File: app/src/main/java/com/leia/leiacam2/CameraFragment.kt [hunk 2]
```diff
@@ -160,6 +180,39 @@ class CameraFragment : Fragment(), CaptureInterface, SwitchInterface, CameraMode
...
+        super.onPause()
+        when (mAudioManager?.ringerMode) {
+            AudioManager.RINGER_MODE_NORMAL -> {
+                mAudioManager?.setStreamVolume(AudioManager.STREAM_MUSIC, mDefaultVolume, AudioManager.FLAG_PLAY_SOUND)
+            }
+        }
+        // Release MediaPlayer if created
+        if (mShutterSoundPlayer != null) {
+            mShutterSoundPlayer?.release()
+        }
```
Comment (Sean7852) [inline app/src/main/java/com/leia/leiacam2/CameraFragment.kt:201]: Better put this in a function, we use the same code in onPause and onDestroy

File: app/src/main/java/com/leia/leiacam2/CameraFragment.kt (hunk 1 above)
Comment (saurabhathavale) [reply app/src/main/java/com/leia/leiacam2/CameraFragment.kt:209]: As we set the stream volume to max = 15 while clicking the photo, need to set it back to default volume when closing the app or pausing the app

File: app/src/main/java/com/leia/leiacam2/CameraFragment.kt (hunk 2 above)
Comment (saurabhathavale) [reply app/src/main/java/com/leia/leiacam2/CameraFragment.kt]: done

**Review by Sean7852 — APPROVED** [review]

**Pull request merged**